app/assets/stylesheets/confirmation_letter.css

```diff
@@ -0,0 +1,8 @@
+/*
+ * This stylesheet will be used only for the printable confirmation letters.
+ *
+ *= require ./confirmation_letter/reset.css
```

Comment:
Given that these two `require`s are commented out – are these stylesheets actually being used? `reset.css` in particular seems to be doing a lot of work that we don't really need it to (styling buttons and hover behaviour, etc)

Reply:
We've discussed this. They're not commented out, this is just sprocket's unusual include snippet syntax.